index.html

```diff
@@ -202,12 +203,12 @@ <h4>Contact Us</h4>
         </p>
       </div>
 
-      <form class="contantform">
-        <input type="text" name="name" id="name" placeholder="Text Input" />
-        <input type="text" name="email" placeholder="mail@example.ir" />
-        <textarea name="text" placeholder="Input text here "></textarea>
+      <form class="contantform" method="post" action="click">
```

Comment:
- [ ] You need to actually set up your formspree account and integrate the provided ID as provided by them.

  ```
  <form class="contantform" method="post" action="https://formspree.io/f/masouma_form_id">
    ...
  ```